final-project/static/scripts.js

```diff
@@ -241,25 +241,82 @@ function editProduct() {
   // Select references.
   // Select ordered list of references.
   var prodPageReferences = document.querySelector('.references-list');
+  // Select all list items in reference list.
+  var referenceList = prodPageReferences.childNodes;
+
+  // Store page elements into a map for easy access.
+  var prodPageElements = new Map();
+
+  prodPageElements.set(name, prodPageName);
```

Comment:
Where is `name` defined?

It's simpler to just use a plain old object here, rather than a Map.

Reply:
I forgot to create variables for all the map elements I set. This code has not been tested yet.

I'll check out the Object option.